roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/StreetSection.java

```diff
@@ -56,8 +64,11 @@ public void moveFirstCarToNextSection() {
 
     }
 
-    @Override
-    public void addCar(ICar car) {
+    public Queue<Car> getQueue() {
+        return queue;
+    }
 
+    public void setQueue(Queue<Car> queue) {
```

Comment:
queue should not be set from outside, it should be managed by the section only

Reply:
+1 consider initializing queue within constructor, also refactor holding property to be **final**.